ion/src/simulator/shared/main_sdl.cpp

```diff
@@ -23,8 +23,8 @@ int main(int argc, char * argv[]) {
 
   char * language = IonSimulatorGetLanguageCode();
   if (language != nullptr) {
-    arguments.push_back("--language");
-    arguments.push_back(language);
+    arguments.insert(arguments.begin(), language);
+    arguments.insert(arguments.begin(), "--language");
```

Comment:
Adding "--language LANGUAGE" option only when the "--language" option is not already present in the arguments would be more robust maybe ? In the case we change how we parse the options in Epsilon.

Reply:
See https://github.com/numworks/epsilon/pull/1641/commits/2c42aa674b8a6f271066d11110f53d1019d17871